README.md

```diff
@@ -6,11 +6,11 @@ We hope you will use this repository as an example of how to integrate Flashbots
 
 Environment Variables
 =====================
-- ETHEREUM_RPC_URL - Ethereum RPC endpoint. Can not be the same as FLASHBOTS_RPC_URL
-- PRIVATE_KEY - Private key for the Ethereum EOA that will be submitting Flashbots Ethereum transactions
-- FLASHBOTS_KEY_ID / FLASHBOTS_SECRET - Flashbots submissions requires an API key. [Apply for an API key here](https://docs.google.com/forms/d/e/1FAIpQLSd4AKrS-vcfW1X-dQvkFY73HysoKfkhcd-31Tj8frDAU6D6aQ/viewform) 
-- HEALTHCHECK_URL _[Optional]_ - Health check URL, hit only after successfully submitting a bundle.
-- MINER_REWARD_PERCENTAGE _[Optional, default 80]_ - 0 -> 100, what percentage of overall profitability to send to miner.
+- **ETHEREUM_RPC_URL** - Ethereum RPC endpoint. Can not be the same as FLASHBOTS_RPC_URL
+- **PRIVATE_KEY** - Private key for the Ethereum EOA that will be submitting Flashbots Ethereum transactions
+- **FLASHBOTS_RELAY_SIGNING_KEY** _[Optional, default: random]_ - Flashbots submissions requires an Ethereum private key to sign transaction payloads. This is NOT a real Ethereum account containing funds, but used to help identify related to same bot. Please see https://hackmd.io/@flashbots/rk-qzgzCD
```

Comment:
"NOT a real Ethereum account" is a bit confusing. It is a real account, but should not have funds